spec/spec_helper.rb

```diff
@@ -1,31 +1,70 @@
+ENV["RAILS_ENV"] ||= 'test'
+ENV["AUTOMATE_DOMAINS"] = "ManageIQ" # Reset only the ManageIQ automate domain when testing.
+
+require File.expand_path("../manageiq/config/environment", __FILE__)
+require 'rspec/rails'
 require 'simplecov'
-SimpleCov.start
 
-require 'manageiq-content'
+SimpleCov.start
```

Comment:
SimpleCov and Coveralls should be required and started before anything is required.